src/main/java/elrond/Transaction.java

```diff
@@ -129,15 +128,24 @@ public String computeHash() {
         return this.txHash;
     }
 
-    private ByteString serializeValue() {
+    public ByteString serializeValue() {
```

Comment:
Could have also been a small component separate from the Transaction component. E.g. `BigIntegerCodec`.

Reply:
good idea. refactored